src/components/Alert/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -16,8 +16,8 @@ exports[`Alert renders correctly unknow theme type and fallback to defaults 1`]
   align-items: center;
   border-radius: 4px;
   padding: 12px;
-  background-color: #ffe1e7;
-  color: #ef5774;
+  background-color: #FFE1E7;
```

Comment:
Why is there some colors on all caps and other on lower ?

Reply:
It comes from figma, I can ask design team to make it uniform

Reply:
We can force a lowercase with jq ?